src/main/java/frc/robot/misc/Constants.java

```diff
@@ -131,9 +136,9 @@ public static final class Mod1 {
         /* Back Left Module - Module 2 */
         public static final class Mod2 {
             public static final int driveMotorID = 9;
-            public static final int angleMotorID = 8;
+            public static final int angleMotorID = 6;//8;
```

Comment:
I don't like these comments at the end of the line anywhere. Please put them above and explain why you're leaving the old value and how you derived the new one or take it out entirely.